migrations/2021/Version20210109123938.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20210109123938 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->connection->beginTransaction();
+        $this->connection->executeStatement('CREATE TABLE ac_camp_cashbooks  (id CHAR(36) NOT NULL COMMENT \'(DC2Type:skautis_camp_id)\',  cashbook_id CHAR(36) NOT NULL COMMENT \'(DC2Type:cashbook_id)\', PRIMARY KEY(id)) DEFAULT CHARACTER SET UTF8 COLLATE `UTF8_unicode_ci` ENGINE = InnoDB');
+        $this->connection->executeStatement('CREATE TABLE ac_event_cashbooks (id CHAR(36) NOT NULL COMMENT \'(DC2Type:skautis_event_id)\', cashbook_id CHAR(36) NOT NULL COMMENT \'(DC2Type:cashbook_id)\', PRIMARY KEY(id)) DEFAULT CHARACTER SET UTF8 COLLATE `UTF8_unicode_ci` ENGINE = InnoDB');
+        $this->savePairs(
+            'ac_event_cashbooks',
+            $this->connection->fetchAllAssociative('SELECT id as cashbook_id, skautisId as id FROM ac_object WHERE type = \'general\'')
+        );
+
+        $this->savePairs(
```

Comment:
Tady by bylo lepší použít jen [INSERT ... SELECT](https://dev.mysql.com/doc/refman/8.0/en/insert-select.html), než to načítat a iterovat nad tím